tools/MemoratorUploader.py

```diff
@@ -38,7 +38,11 @@ def upload(log_file: kvmlib.LogFile, parserCallFunc: callable, live_filters: lis
             timestamp = start_time + float(match.group(1))
             timestamp_str = struct.pack('>d', timestamp).decode('latin-1')
 
-            id = int(match.group(3), 16)
+            id = int(match.group(3).strip(), 16)
+
+            if id == 0:
```

Comment:
Can you clarify `id == 0`

Reply:
Good question. This is from the memorator actually. Sometimes it gives id = 0. We believe this is an error frame so if we get this ID no point in parsing it so we continue the loop/skip it. Note we think its an error frame because the data of an error frame matches exactly that of the id = 0 messages.
![image](https://github.com/UBC-Solar/sunlink/assets/117491745/649aef50-cd95-459b-ba15-0b6550fb05a7)

Reply:
Can you make it a macro like `ERROR_ID` or something?